doc/user/bgp.rst

```diff
@@ -2918,7 +2918,20 @@ Extended Community Lists
 BGP Extended Communities in Route Map
 """""""""""""""""""""""""""""""""""""
 
-.. clicmd:: match extcommunity WORD
+.. clicmd:: match extcommunity WORD [exact-match]
+
+   This command perform match to BGP updates using extended community list WORD.
+   When the one of BGP extended communities value match to the one of the extended
+   communities value in community list, it is match. When `exact-match` keyword is
```

Comment:
To highlight, we should use double ticks.

Reply:
ok